buildah.go

```diff
@@ -259,6 +261,8 @@ type BuilderOptions struct {
 	FromImage string
 	// Container is a desired name for the build container.
 	Container string
+	// ContextDirectory of current build
+	ContextDir string
```

Comment:
Make this a field in `RunOptions`, so that it can differ for different `buildah run` invocations, like the `--contextdir` function that `buildah add` and `buildah copy` provide.

Reply:
ah makes sense thanks will do changes.

Reply:
Added override so --contextDir from `run` could be used if specified
```go
func (b *Builder) Run(command []string, options RunOptions) error {
         // override contextdir from RunOptions if specified explicitly via --contextDir
	// otherwise use ContextDir configured from build
	if options.ContextDir != "" {
		b.ContextDir = options.ContextDir
	}

```